Pspcl/Pspcl.Services/Class1.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Pspcl.Services
+{
+    public class Class1
+    {
```

Comment:
Why this class is added.?

Reply:
by default it was added. I'll remove it now.